control-plane/pkg/reconciler/channel/resources/service.go

```diff
@@ -28,8 +30,12 @@ import (
 )
 
 const (
-	portName           = "http"
-	portNumber         = 80
+	portName   = "http"
+	portNumber = 80
+
+	TLSportName   = "https"
+	TLSportNumber = 443
```

Comment:
```suggestion
	tlsPortName   = "https"
	tlsPortNumber = 443
```
Can we make these variables private, like the non-tls variables?